CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Improvements
+- **UI:** Improve search result ordering. The search result will be boosted to
```

Comment:
As we just discussed, we should write something about priority here. Something like (maybe you find a better wording):
... The search results will be ordered by the priority of the fields matching the query. The priority is like the following name -> description -> tenant/repo.

Reply:
will fix.